module/applications/source-config.mjs

```diff
@@ -35,7 +34,7 @@ export default class SourceConfig extends DocumentSheet5e {
 
   /** @override */
   get title() {
-    return game.i18n.localize("DND5E.SourceConfig");
+    return game.i18n.localize("DND5E.SOURCE.Action.Configure");
```

Comment:
Since `window.title` is localized automatically, is this getter even needed?

Reply:
This is because we are expanding `DocumentSheetV2`, which provides its own title getter that uses the document's name instead.